modules/core/ui/metrics/src/Plugin/Block/FarmMetricsBlock.php

```diff
@@ -74,7 +74,7 @@ public function build() {
 
     // Create a container for asset metrics.
     $output['asset'] = [
-      '#markup' => '<strong>' . Link::createFromRoute('Assets', 'view.farm_asset.page')->toString() . '</strong>',
+      '#markup' => '<strong>' . Link::createFromRoute('Assets', 'view.farm_asset.page')->toString() . ' (active)</strong>',
```

Comment:
We should translate `(active)`, and also likely `Assets` (and `Logs` below).

This might be related to this report: https://farmos.discourse.group/t/farmos-2-0-translation-localization-l10n-initiative/922/7?u=paul121